python/katana/local/_graph.pyx

```diff
@@ -275,6 +278,28 @@ cdef class GraphBase:
         """
         return str(self.underlying_property_graph().rdg_dir(), encoding="UTF-8")
 
+    @property
+    def node_types(self):
+        """
+        The types of atomic node types in the graph.
+
+        :rtype: list
+        """
+        cdef const EntityTypeManager* manager = &self.underlying_property_graph().GetNodeTypeManager()
```

Comment:
NIT: I don't think you need the type here. I think it can infer it.

Reply:
The compiler actually complains when I remove it about some kind of invalid conversion. It does seem necessary here.

Reply:
This is an example where the convention of field accessors being snake_case is helpful because you know that you can take an address of a field, but if `GetNodeTypeManager` were to return a value rather than a reference, we'd have problems.